### ISSUE TITLE
Launchers on MATE panel at top of screen flicker when trying to display tooltips

### ISSUE THREAD

**Issue by smkent** [mate-desktop/mate-panel#677]
#### Steps to reproduce the behaviour

1) Install Linux Mint 18.2 MATE.
2) Move the default panel from the bottom of the screen to the top of the screen.
3) Hover the mouse cursor over one of the launchers.
4) Observe the launcher's tooltip rapidly appears and disappears, while the launcher icon flickers. (Expected behavior: The tooltip should appear normally.) This also occurs sometimes on the mintmenu applet.

This happens a majority of the time on a majority of the launchers, but not every time on every launcher. I moved my panel back down to the bottom and the issue does not reproduce. Looking closely, the tooltips that rapidly appear/disappear are being rendered with different sizes/shapes so it's almost as though the tooltip size/position calculator is having trouble with the upper edge of the screen.

#### MATE general version

1.18.0

#### Linux Distribution

Linux Mint 18.2 MATE

**Comment by smkent** [comment]
This also reproduces in the Linux Mint 18.2 MATE install CD's live environment. Simply boot up the CD, move the default panel to the top, and hover over the launchers. The same behavior occurs, sometimes the tooltips appear and sometimes they don't. I tried the live media in a VM, and in this case the launcher icons didn't flicker as visibly but it still happens if you look closely.

I also booted up the Linux Mint 18.1 MATE (MATE version 1.16.1) install CD in another VM, and confirmed the issue does NOT reproduce in that environment. Something changed between MATE version 1.16.1 and 1.18.0 that introduced this bug.
